src/components/Table/Tests/__snapshots__/Table.pagination.renders.shot

```diff
@@ -2209,7 +2209,7 @@ LoadedCheerio {
                           "aria-checked": "true",
                           "aria-disabled": "false",
                           "aria-pressed": "true",
-                          "class": "pagination-button active button button-neutral button-medium",
+                          "class": "pagination-button active button button-neutral button button-medium",
```

Comment:
why did the additional and highly generic button class get added? is this desirable? does it get hashed?

Reply:
Needed to add this because the button doesn't have a wrapping element.